src/main/java/org/littleshoot/proxy/impl/ClientToProxyConnection.java

```diff
@@ -22,18 +22,17 @@
 import io.netty.handler.traffic.GlobalTrafficShapingHandler;
 import io.netty.util.concurrent.Future;
 import io.netty.util.concurrent.GenericFutureListener;
-
```

Comment:
I agree, IntelliJ reformatted the includes automatically without me telling it to. It would be especially nice to get a default column width of 120 or 140. There's a lot of code that wraps way too early.